### ISSUE TITLE
duplicate symbols for architecture armv7

### ISSUE THREAD

**Issue by lefeverela** [ShareKit/ShareKit#1045]
The problem is the following: ShareKit is using SDWebImage, and the current project I am importing it too, also have SDWebImage. So the compilator is not happy because he see 2 times the library. Once within ShareKit and once within my project, so I cannot compile.

If I remove the SDWebImage from my project, then I got compilation errors like it cannot find SDWebManager.h and other differents class of the SDWebImage package I am using. How should I setup my project so that I only have one SDWebImage package used by both and that allow me to make ShareKit works, I am quite noobs in all those setup things which is so much painful that the actual coding.

My error message below FYI:

ld: 85 duplicate symbols for architecture armv7

duplicate symbol _OBJC_IVAR_$_SDWebImagePrefetcher._manager in:
    /Users/admin/Library/Developer/Xcode/DerivedData/TMO-grzsoumetfbfmobxapaqwbryuafg/Build/Intermediates/TMO.build/Debug-iphoneos/TMO.build/Objects-normal/armv7/SDWebImagePrefetcher.o
    /Users/admin/Library/Developer/Xcode/DerivedData/TMO-grzsoumetfbfmobxapaqwbryuafg/Build/Products/Debug-iphoneos/libShareKit.a(SDWebImagePrefetcher.o)
duplicate symbol _OBJC_IVAR_$_SDWebImagePrefetcher._options in:
    /Users/admin/Library/Developer/Xcode/DerivedData/TMO-grzsoumetfbfmobxapaqwbryuafg/Build/Intermediates/TMO.build/Debug-iphoneos/TMO.build/Objects-normal/armv7/SDWebImagePrefetcher.o
    /Users/admin/Library/Developer/Xcode/DerivedData/TMO-grzsoumetfbfmobxapaqwbryuafg/Build/Products/Debug-iphoneos/libShareKit.a(SDWebImagePrefetcher.o)
duplicate symbol _OBJC_IVAR_$_SDWebImagePrefetcher._prefetcherQueue in:

.....

**Comment by lefeverela** [comment]
After spending few hours on this I think I found the solution. This is what I did:
1) Remove SDWebImage package from ShareKit as well as libShareKit.a
2) Add the whole SDWebImage folder to ShareKit project adding files to ShareKit
3) Removing the SDWebImage library from my master project
4)The go in the header path (in build settings) and add the folter where ShareKit is installed and select recursive so that it will look for all the header in the ShareKit folder and find SDWebImage.

I think that's all I did.
